spotless-maven-pom.xml

```diff
@@ -26,6 +26,9 @@
                             <includes>
                                 <include>**/*.java</include>
                             </includes>
+                            <excludes>
+                                <exclude>**/non_formatted_code/*</exclude>
+                            </excludes>
```

Comment:
This was necessary because this file would otherwise be formatted by spotless when running pre-commit with the `--all-files` flag.

Since spotless doesn't receive individual files from pre-commit, the pre-commit exclude list doesn't apply.